website/docs/cloud-docs/api-docs/agents.mdx

```diff
@@ -513,9 +513,9 @@ Properties without a default value are required.
 |---------------------------------------------------|--------|------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `data.type`                                       | string |                  | Must be `"agent-pools"`.                                                                                                                                                                                                                         |
 | `data.attributes.name`                            | string | (previous value) | The name of the agent pool, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.                                                                                    |
-| `data.attributes.organization-scoped`             | bool   | true             | The scope of the agent pool. If true, all workspaces in the organization can use the agent pool. Available in Terraform Enterprise v202207-1 and later.                                                                                          |
-| `data.relationships.allowed-workspaces.data.type` | string |                  | Must be `"workspaces"`. Available in Terraform Enterprise v202207-1 and later.                                                                                                                                                                   |
-| `data.relationships.allowed-workspaces.data.id`   | string |                  | The ID of the workspace that is allowed to use the agent pool. Refer to [Scoping Agent Pools to Specific Workspaces](/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). Available in Terraform Enterprise v202207-1 and later. |
+| `data.attributes.organization-scoped`             | bool   | true             | The scope of the agent pool. If true, all workspaces in the organization can use the agent pool.                                                                                          |
+| `data.relationships.allowed-workspaces.data.type` | string |                  | Must be `"workspaces"`.                                                                                                                                                                   |
+| `data.relationships.allowed-workspaces.data.id`   | string |                  | The ID of the workspace that is allowed to use the agent pool. Refer to [Scoping Agent Pools to Specific Workspaces](/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). Available in  and later. |
```

Comment:
```suggestion
| `data.relationships.allowed-workspaces.data.id`   | string |                  | The ID of the workspace that has permission to use the agent pool. Refer to [Scoping Agent Pools to Specific Workspaces](/cloud-docs/agents#scope-an-agent-pool-to-specific-workspaces). |
```

Suggestion to change from "is allowed to" to "has permission to" for passive voice.

Also suggest removing "Available in  and later" or else putting in a different available release in place of the now removed one.

Reply:
thank you for catching this!!! in my effort to move quickly this "available in and later" is a complete mistake. Thank you!